hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/MetricsReservoirType.java

```diff
@@ -0,0 +1,5 @@
+package pl.allegro.tech.hermes.common.metric;
+
+public enum MetricsReservoirType {
```

Comment:
WDYT to move this to `MetricsReservoirType` ?

Reply:
I would like to be consistent with the current implementation of similar case: `pl.allegro.tech.hermes.common.schema.SchemaRepositoryType`.